### ISSUE TITLE
review and update final project rubric

### ISSUE THREAD

**Issue by heratyian** [DPI-WE/rubric#4]
I want to make some updates to the final project rubric based on experience from last cohort. Can @DPI-WE/teaching-assistants and @DPI-WE/staff  take a look and submit feedback? Either comment on this issue or create a pull request with notes. Anytime over the next couple weeks is fine.

**Comment by heratyian** [comment]
@DPI-WE/teaching-assistants can we review the rubric this week? Any and all feedback welcome. 🙏

**Comment by flealc** [comment]
Looks great!

There is a weird indentation for the checkboxes [here](https://github.com/DPI-WE/rubric/blob/4cd2ebe086dab75312c51dae8b2a81d3642ff6eb/sdf/final-project/business.md?plain=1#L14) but other than that the rubric itself makes perfect sense to me

**Comment by thierrychau** [comment]
A few observations:
- Are graders expected to read each sub-rubric (example: "Documentation") and give an overall grade for it (example: trainee is hitting ~80% so 4/5)? 
or is each bullet point in each sub-rubric supposed to represent 1 point? 
I think the latter could help with expectations and also homogenize grades amongst staff. 
However, for some rubrics with more bullet points (example: Code Hygiene) or which don't have a common denominator (example: 8 bullet points for 5 points), it could need some tuning.
tldr: indicate how much point is each bullet point worth
- What do you think of making the current "bonuses" which we can designate as part of the expectations? 
As a trainee, I considered anything written down, even if labelled as a bonus, as required. My train of thought was that these were "easy" points that the staff have identified, and not trying to get them is a missed opportunity. (Example: bonus point for getting a custom domain => go ahead and do it!)
- Keep the bonus section for things we have truly not anticipated. Maybe each grader is only allowed to give out 5 times 2 bonus points or some kind of rule?

I'm going to review more specifically the weight of each sub-rubric and comment back here.

**Comment by ajmompr** [comment]
My only feedback is regarding the weight of the 3 sections: technical, business, and presentation. Would you be open to reducing the weight of the presentation section? For our cohort, the presentation portion of the project seemed to invite the most subjective feedback and may also penalize trainees who are not the most outspoken.

**Comment by flealc** [comment]
> My only feedback is regarding the weight of the 3 sections: technical, business, and presentation. Would you be open to reducing the weight of the presentation section? For our cohort, the presentation portion of the project seemed to invite the most subjective feedback and may also penalize trainees who are not the most outspoken.

I tend to agree. Presentations skills could still weigh in order to get great/top score, but assessing them does invite the subjective component that @ajmompr mentioned, potentially introducing inconsistencies in the grading results.

**Comment by CarlosCastanedaDev** [comment]
I think we should rephrase how bonus points are applied. 5 points per section seems like a lot, 15 bonus points total seems a bit excessive.

**Comment by CarlosCastanedaDev** [comment]
I like @thierrychau idea of limiting how many bonus points can be given by a grader. "Keep the bonus section for things we have truly not anticipated. Maybe each grader is only allowed to give out 5 times 2 bonus points or some kind of rule?'

**Comment by thierrychau** [comment]
> My only feedback is regarding the weight of the 3 sections: technical, business, and presentation. Would you be open to reducing the weight of the presentation section? For our cohort, the presentation portion of the project seemed to invite the most subjective feedback and may also penalize trainees who are not the most outspoken.

I do think any professional should be able to present for 3 min on a project they have worked on for weeks, especially in a relatively closed setting since only staff is present for the grading portion. In the bigger picture, 20 points is less than 20% of their total grade (including the business readiness). Someone who bombs a presentation can still get 80%+.

To fight inconsistencies, can we explore adding more details to each sub-rubric to leave less room for interpretation?

**Comment by ajmompr** [comment]
> > My only feedback is regarding the weight of the 3 sections: technical, business, and presentation. Would you be open to reducing the weight of the presentation section? For our cohort, the presentation portion of the project seemed to invite the most subjective feedback and may also penalize trainees who are not the most outspoken.
> 
> I do think any professional should be able to present for 3 min on a project they have worked on for weeks, especially in a relatively closed setting since only staff is present for the grading portion. In the bigger picture, 20 points is less than 20% of their total grade (including the business readiness). Someone who bombs a presentation can still get 80%+.
> 
> To fight inconsistencies, can we explore adding more details to each sub-rubric to leave less room for interpretation?

Fair point; adding details would be an easy work around that won't require changing the weight of each section. I do think it is important to stress to trainees that the presentation will be a significant portion of their final project grade. Also, we should aim to get the rubric in front of them as early as possible to avoid any surprises!
